tests/chart_tests/test_astronomer_pod_security_contexts.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+from tests import git_root_dir
+from tests.chart_tests.helm_template_generator import render_chart
+
+include_kind_list = ["Deployment", "DaemonSet", "StatefulSet", "ReplicaSet"]
+
+default_podsecuritycontext = {
+    "fsGroup": 1000,
+    "runAsGroup": 1000,
+    "runAsUser": 1000,
+}
```

Comment:
We should not use the same values as the defaults. We should use something obviously custom, like 998877.